src/main/java/soconnect/model/todo/UniqueTodoList.java

```diff
@@ -22,29 +27,30 @@ public class UniqueTodoList implements Iterable<Todo> {
         FXCollections.unmodifiableObservableList(internalList);
 
     /**
-     * Returns true if the list contains an equivalent todo as the given argument.
+     * Returns true if the list contains an equivalent {@code Todo} as the given argument.
      */
     public boolean contains(Todo toCheck) {
         requireNonNull(toCheck);
         return internalList.stream().anyMatch(toCheck::equals);
     }
 
     /**
-     * Adds a todo to the list.
-     * The todo must not already exist in the list.
+     * Adds a {@code Todo} to the list.
+     * The {@code Todo} must not already exist in the list.
      */
     public void add(Todo toAdd) {
         requireNonNull(toAdd);
         if (contains(toAdd)) {
             throw new DuplicateTodoException();
         }
         internalList.add(toAdd);
+        sort();
     }
 
     /**
-     * Replaces the todo {@code target} in the list with {@code editedTodo}.
+     * Replaces a {@code Todo} in the list with {@code editedTodo}.
      * {@code target} must exist in the list.
-     * The new todo must not be the same as an existing todo in the list.
+     * {@code editedTodo} must not be the same as an existing {@code Todo} in the list.
```

Comment:
```suggestion
     * The {@code editedTodo} must not be the same as an existing {@code Todo} in the list.
```

Just trying to be safe and start the sentence with an uppercase letter.